modules/installing-gcp-user-defined-labels-and-tags.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+// * installing/installing_gcp/installing-gcp-customizations.adoc
+
+:_content-type: CONCEPT
+[id="installing-gcp-user-defined-labels-and-tags_{context}"]
+= Configuring the user-defined labels and tags for GCP
+
+
+Google Cloud Platform (GCP) provides `labels` and `tags` that help to identify and organize the resources created for a specific {product-title} cluster, making them easier to manage. 
+
+You can define `labels` and `tags` for each GCP resource only during {product-title} cluster installation. 
+
+[IMPORTANT]
+====
+User-defined `labels` and `tags` are not supported for {product-title} clusters upgraded to {product-title} 4.14 version.
+====
+
+.User-defined labels
+
+User-defined `labels` and {product-title} specific labels are applied only to resources created by {product-title} installer and its core operators such as: 
+* `machine-api-provider-gcp` 
+* `cluster-image-registry-operator`
+* `gcp-filestore-csi-driver-operator`
+* `gcp-pd-csi-driver-operator` 
+
+The user-defined `labels` and {product-title} specific labels are not applied on the resources created by any other operators or the kubernetes in-tree components that create resources, for example, the Ingress load balancers.
+
+The following are the requirements for the user-defined labels:
+
+* A label key and value must have a minimum of 1 character and can have a maximum of 63 characters.
+* A label key and value must contain only lowercase letters, numeric characters, underscore (`_`), and dash (`-`).
+* A label key must start with a lowercase letter.
+* You can configure a maximum of 32 labels per resource. Each resource can have a maximum of 64 labels, and 32 labels are reserved for internal use by {product-title}.
+
+.Limitations for user-defined labels
+
+* Labels for `Compute Address` are supported in the GCP beta version. {product-title} does not add labels to the resource.
+
+User-defined labels and {product-title} labels are seen on the following GCP resources:
+
+* Compute Instance
+* Compute Disk
+* Compute Forwarding Rule
+* Compute Image
+* DNS Managed Zone
+* Filestore Instance
+* Storage Bucket
+
+.User-defined tags
+
+User-defined `tags` are attached to resources created by {product-title} core operator `cluster-image-registry-operator` and not on the resources created by any other operators or the kubernetes in-tree components. In {product-title} version {product-version}, resources created by the foloowing operators are not tagged:
```

Comment:
Also change the Operator name as suggested by Jeana.

```suggestion
User-defined tags are attached to resources created by {product-title} core operator `cluster-image-registry-operator` and not on the resources created by any other operators or the kubernetes in-tree components. In {product-title} version {product-version}, resources created by the following operators are not tagged:
```

Reply:
The operators name are changed as per Jeana's suggestion. Earlier they were mentioned as `machine-api-provider-gcp`, etc.

Reply:
I think Shuba is referring to `cluster-image-registry-operator` here:
```suggestion
User-defined tags are attached to resources created by the {product-title} Cluster Image Registry Operator and not on the resources created by any other Operators or the Kubernetes in-tree components. In {product-title} version {product-version}, resources created by the following Operators are not tagged:
```
Also, note: still seeing "tags" and "labels" in backticks throughout this PR, and the word Operator in lower case (should be capitalized).